website/docs/install/install-with-script.md

```diff
@@ -74,23 +69,22 @@ If you don't have 32 ETH to stake, <a href='https://ethereum.org/en/staking/pool
 - **Try this guide on testnet first**, *then* mainnet.
 - **Keep things simple**. This guidance assumes all client software will run on a single machine.
 - **Review the latest advisories** for both [testnet](https://prater.launchpad.ethereum.org/en/overview) and [mainnet](https://launchpad.ethereum.org/en/).
-- **Review all of our [published security best practices](./security-best-practices/)**.
+- **Review** our [published security best practices](../security-best-practices.md).
 - **Join the community** - join our [mailing list](https://groups.google.com/g/prysm-dev), the [Prysm Discord server](https://discord.com/invite/XkyZSSk4My), [r/ethstaker](https://www.reddit.com/r/ethstaker/), and the [EthStaker Discord server](https://discord.io/ethstaker) for updates and support.
 
 
 
 ## Step 3: Run an execution client
 
+You need two different pieces of software to run an Ethereum node: an **execution-layer client** and a **consensus-layer client**. Prysm is a consensus-layer client that contains code for both beacon node and validator responsibilities. In this step, you'll install an execution-layer client that Prysm will use [<a href='#footnote-2'>2</a>].
```

Comment:
Can we also briefly explain what an execution client is? Maybe we can just say it is responsible for processing user transactions and smart contracts

Reply:
The footnote explains this - or are you asking to pull that up here?

Reply:
Up here is good because if offers enough context without needing to navigate away into the footnote section. Just a simple explanation is enough